src/config/tablet.c

```diff
@@ -7,6 +7,17 @@
 #include "config/tablet.h"
 #include "config/rcxml.h"
 
+double
+tablet_get_area_value(const char *content, const char *name)
```

Comment:
Consider parsing percentages, for example 50%

Reply:
That would indeed be cool, but i don't have the tablet dimensions available here. If we allow this, we would need to do the conversion from relative to absolute when doing the actual conversion in https://github.com/labwc/labwc/blob/27bba861e85283cc7619e512fd0b300ead0d5257/src/input/tablet.c#L15 
We would also need to store if width/height are absolute of relative. This won't fit into `wlr_fbox` .  I would prefer to not do this as part of this PR and eventually come back to this later.

Reply:
Maybe one for the future :smile: